### PR TITLE
Bugfix/subscription port

### PR DESCRIPTION
**Pull request by krauskopf**

### CODE REVIEW

File: lib/CtrlxDatalayerSubscription.js [hunk 1]
```diff
@@ -197,7 +199,7 @@ class CtrlxDatalayerSubscription extends EventEmitter {
       this._es.onerror = (e) => {
         debug(`onerror(${e.type})`);
 
-        if (e.data !== 'undefined') {
+        if (typeof e.data !== 'undefined') {
```
Comment (ZmRZjFuDsDiR4qi5801F) [inline lib/CtrlxDatalayerSubscription.js:202]: arggh

File: lib/CtrlxCore.js [hunk 2]
```diff
@@ -807,7 +807,7 @@ class CtrlxCore {
       }
 
       // The subscription is encapsulated in a separate object.
-      let sub = new CtrlxDatalayerSubscription(this._hostname, this._servername, this._tokenType + ' ' + this._token, paths, publishIntervalMs);
+      let sub = new CtrlxDatalayerSubscription(this._hostname, this._port, this._servername, this._tokenType + ' ' + this._token, paths, publishIntervalMs);
```
Comment (ZmRZjFuDsDiR4qi5801F) [inline lib/CtrlxCore.js:810]: 1. Is there any option to set the port in the Editor by the user? I cannot find any either on the subscription or the device config node. 
2. How can I setup the tests to be executed using a different port than 443?

**Review by ZmRZjFuDsDiR4qi5801F — APPROVED** [review]
Looks fine, thank you for answering the questions.

File: lib/CtrlxCore.js (hunk 2 above)
Comment (krauskopf) [reply lib/CtrlxCore.js:810]: 1. You can set the port by appending in the hostname. E.g. "192.168.1.1:8443"
2. You can change the hostname and port for the unit test by setting the environment variable TEST_HOSTNAME before the test. E.g. "export TEST_HOSTNAME=localhost:1234"

**Pull request merged**